### PR TITLE
GITLAB-1123 Scenario exercising only default INBOX search

### PR DESCRIPTION
**Pull request by chibenwa**
Exercises only GetMessageList default inbox view to mesure ElasticSearch default search performance.

### CODE REVIEW

**Comment by chibenwa** [comment]
```
[error] /home/travis/.sbt/1.0/staging/05cb89211ea66aea052a/gatling-imap/src/main/scala-2.12/com/linagora/gatling/imap/protocol/command/AppendHandler.scala:31:21: value foreach is not a member of java.util.stream.Stream[String]

[error]       content.lines.foreach(executeTextCommand(session))

[error]                     ^

[error] /home/travis/.sbt/1.0/staging/05cb89211ea66aea052a/gatling-imap/src/main/scala-2.12/com/linagora/gatling/imap/protocol/command/AppendHandler.scala:31:47: missing argument list for method executeTextCommand in class AppendHandler
```

Don't seems related with my work

**Review by Arsnael — APPROVED** [review]

File: src/it/scala-2.12/org/apache/james/gatling/DefaultInboxSearchLoadingIT.scala [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package org.apache.james.gatling
+
+import org.apache.james.gatling.jmap.scenari.DefaultInboxSearchLoadingScenario
+
+class DefaultInboxSearchLoadingIT extends JmapIT {
+
+  before {
+    users.foreach(server.sendMessage(Fixture.homer.username))
```
Comment (Arsnael) [inline src/it/scala-2.12/org/apache/james/gatling/DefaultInboxSearchLoadingIT.scala]: Actually you don't miss the `org.apache.james.gatling.Fixture` import for that?

**Comment by remk** [comment]
```
error] /home/travis/.sbt/1.0/staging/05cb89211ea66aea052a/gatling-imap/src/main/scala-2.12/com/linagora/gatling/imap/protocol/command/AppendHandler.scala:31:21: value foreach is not a member of java.util.stream.Stream[String]

[error]       content.lines.foreach(executeTextCommand(session))

[error]                     ^

[error] /home/travis/.sbt/1.0/staging/05cb89211ea66aea052a/gatling-imap/src/main/scala-2.12/com/linagora/gatling/imap/protocol/command/AppendHandler.scala:31:47: missing argument list for method executeTextCommand in class AppendHandler

[error] Unapplied methods are only converted to functions when a function type is expected.

[error] You can make this conversion explicit by writing `executeTextCommand _` or `executeTextCommand(_)(_)` instead of `executeTextCommand`.

[error]       content.lines.foreach(executeTextCommand(session))

[error]                                               ^

[error] /home/travis/.sbt/1.0/staging/05cb89211ea66aea052a/gatling-imap/src/main/scala-2.12/com/linagora/gatling/imap/protocol/command/AppendHandler.scala:41:51: value length is not a member of java.util.stream.Stream[String]

[error]       val length = content.length + content.lines.length - 1

[error]                                                   ^

[error] three errors found

[error] (ProjectRef(uri("git://github.com/linagora/gatling-imap.git"), "gatling-imap") / Compile / compileIncremental) Compilation failed

[error] Total time: 56 s, completed Oct 15, 2019, 3:42:07 AM

restoring stty: 500:5:bf:8a3b:3:1c:7f:15:4:0:1:0:11:13:1a:0:12:f:17:16:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0

The command "sbt compile" exited with 1.
```

**Comment by chibenwa** [comment]
As far as I know  we did not touch the imap append handler in this changeset.

This seems unrelated...

**Comment by chibenwa** [comment]
As far as I know  we did not touch the imap append handler in this changeset.

This seems unrelated...

**Comment by chibenwa** [comment]
The error messages looks like they are generated from gatling/imap project

**Comment by chibenwa** [comment]
See https://github.com/linagora/gatling-imap/pull/40

**Comment by Arsnael** [comment]
```
[error] /home/travis/build/linagora/james-gatling/src/it/scala-2.12/org/apache/james/gatling/DefaultInboxSearchLoadingIT.scala:5:43: not found: type JmapIT

[error] class DefaultInboxSearchLoadingIT extends JmapIT {

[error]                                           ^

[error] /home/travis/build/linagora/james-gatling/src/it/scala-2.12/org/apache/james/gatling/DefaultInboxSearchLoadingIT.scala:7:3: not found: value before

[error]   before {

[error]   ^

[error] /home/travis/build/linagora/james-gatling/src/it/scala-2.12/org/apache/james/gatling/DefaultInboxSearchLoadingIT.scala:8:5: not found: value users

[error]     users.foreach(server.sendMessage(Fixture.homer.username))

[error]     ^

[error] /home/travis/build/linagora/james-gatling/src/it/scala-2.12/org/apache/james/gatling/DefaultInboxSearchLoadingIT.scala:8:19: not found: value server

[error]     users.foreach(server.sendMessage(Fixture.homer.username))

[error]                   ^

[error] /home/travis/build/linagora/james-gatling/src/it/scala-2.12/org/apache/james/gatling/DefaultInboxSearchLoadingIT.scala:11:3: not found: value scenario

[error]   scenario(feederBuilder => {

[error]   ^

[error] 5 errors found

[error] (IntegrationTest / compileIncremental) Compilation failed
```
That one is related :)

**Comment by Arsnael** [comment]
Was a rebase issue, your branch was behind master, seems the hierarchy of IT tests has changed

**Review by remk — APPROVED** [review]

**Pull request merged**